src/common-components/MediumScreenHeader.jsx

```diff
@@ -9,8 +9,8 @@ const MediumScreenHeader = (props) => {
   const { intl } = props;
 
   return (
-    <>
-      <div className="medium-screen-header mb-4">
+    <div className="container row p-0 m-0 medium-screen-container">
+      <div className="col-md-10 p-0 medium-screen-header mb-4">
```

Comment:
You can use bootstrap class `bg-primary` instead creating custom class `medium-screen-header`